meta-lmp-base/recipes-core/initrdscripts/initramfs-framework/cryptfs_pkcs11

```diff
@@ -0,0 +1,54 @@
+# Copyright (C) 2022 Fondries.IO
+# SPDX-License-Identifier: MIT
+
+cryptfs_check_pkcs11() {
+	[ ! -c /dev/tee0 ] && fatal "Linux OP-TEE device node not found"
+}
+
+cryptfs_pre_pkcs11() {
+	# Use TEE Identity with user authentication - root
+	export CKTEEC_LOGIN_TYPE=user
+
+	# tee-supplicant is needed for OP-TEE related operations
+	start-stop-daemon -q -S -b /usr/sbin/tee-supplicant
+}
+
+cryptfs_post_pkcs11() {
+	start-stop-daemon -q -K -x /usr/sbin/tee-supplicant
+}
+
+cryptfs_enroll_pkcs11() {
+	root_dev=$1
+
+	MODULE=/usr/lib/libckteec.so.0
+	# We force usage of the second slot (1) to avoid clashes with lmp-device-register
+	pkcs11-tool --module $MODULE --init-token --slot 1 --label lmp --so-pin ""
+	pkcs11-tool --module $MODULE --token-label lmp --init-pin --so-pin foo --pin user:ef0180a3-13f3-5d1c-b7f5-888c0e21059a
```

Comment:
Maybe I missed something but why do we need to set a pin ?
Is there a particular for this specific value ?

Reply:
We always need a pin, in this case we're using the UID authentication support in OP-TEE (as root), see https://github.com/OP-TEE/optee_client/pull/236.